content/participants/ttjc@Ujjwal-here/index.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Ujjwal-here
+type: "participant"
+name: "Ujjwal Maurya"
+portfolio: ""
+github-link: "https://github.com/Ujjwal-here"
+linkedin-link: "https://www.linkedin.com/in/ujjwal-maurya-94754a191/"
+looking-for: ""
```

Comment:
Please fill the looking for section either **job** or **internship**

Reply:
I have made the necessary changes. Please check.